README.md

```diff
@@ -1,8 +1,9 @@
-# EctoJsonapi
+## Ecto. Json.  Api.
```

Comment:
TIL that Jsonapi is actually `JSON:API`. We'll wanna use that format to keep it consistent with the http://jsonapi.org style.